client/viame-web-common/components/RunPipelineMenu.vue

```diff
@@ -43,10 +43,13 @@ export default defineComponent({
     });
 
     const pipelinesNotRunnable = computed(() => (
-      props.selectedDatasetIds.length > 1 || pipelines.value === null
```

Comment:
As best I can tell, this was just an uncaught typo.  The logic makes no sense otherwise.

Reply:
Might of been to prevent a user from running pipelines on multiple datasets, but looks like a mistake.